### ISSUE TITLE
Design Intent - Accessibility Feedback - CTO Health Team, MHV on VA.gov Portal, Authenticated Landing Page

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#53770]
## VFS acceptance criteria
- [ ] Let Platform know when the **Must** feedback has been incorporated 
- [ ] Leave any comments for feedback you decide _not_ to take
- [ ] VFS team closes the ticket

## Thoughts/questions
- I'm a huge fan of the bigger text size and the extra padding between columns in the three-column section. Small things that help a ton and I love it.

## Feedback
- Practice areas will document their feedback following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback).

**Must:**
- If the links pointing back to the legacy MyHealtheVet application end up opening in a new tab, then at a minimum we'll want to make sure screen reader users are warned about that behavior, eg. `<a href="..." aria-label="Lab and test results (opens in a new tab)">Lab and test results</a>`. Ideally, I would love to see visible text notifying all users about this behavior as opening in a new tab can increase cognitive load and potentially be disorienting for screen magnification users. That's probably a bigger ask in terms of design though.

**Should:**
- Active links with no underline is a departure from [what's currently in the design system](https://design.va.gov/components/link/#active), and generally I would say links should be underlined with very limited exceptions. The use of the chevron may be sufficient for meeting [1.4.1](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html). But I'll still point toward Adrian Roselli's collection of [underline-related references](https://adrianroselli.com/2016/06/on-link-underlines.html), and I'd be more enthusiastic about using this as an opportunity to experiment with `text-underline-position` or `text-underline-offset`.

**Consider:**
- Any time there's a multi-column design, I'm a little concerned about the impact on screen magnification users (particularly users who use magnification software to magnify a portion of the screen, as opposed to those who use the native `Ctrl +`/`Cmd +` browser zoom). I think what's proposed here is probably going to be fine, since there are enough visual elements that should nudge users to explore the right-hand side of their screen. But consider validating that through user research if possible.

## Platform directions
- Remove the governance-team label
- Update "Issue Title"
- Link to collab cycle Request epic
- Add your feedback
- Remove the CC-Request label (only if you created the ticket using the New Issue link from the Collaboration Cycle Reviews board)
- Add assignees based on collab cycle touchpoint
  - **Design Intent**: VFS designer, VFS PM (optional), yourself (optional)
  - **Midpoint Review**: VFS PM, yourself (optional)
  - **Staging Review QA Only**: VFS PM, yourself (optional)

**Comment by MarciMcGuire** [comment]
@dcloud @rtwell -have the issues mentioned in this design intent been resolved?  Thank you!
